src/plugins/tts/plugin.tts.js

```diff
@@ -158,18 +158,31 @@ BookReader.prototype.initNavbar = (function (super_) {
 
       $el.find('.BRcontrols').prepend(this.refs.$BRReadAloudToolbar);
 
+      const renderVoiceOption = (voices) => {
+        return voices.map(voice =>
+          `<option value="${voice.voiceURI}">${voice.lang} - ${voice.name}</option>`).join('');
+      };
+
+      const sortedOrder = () => {
+        return (a,b) => `${a.lang} - ${a.name}`.localeCompare(`${b.lang} - ${b.name}`);
+      };
+
       const renderVoicesMenu = (voicesMenu) => {
         voicesMenu.empty();
+        const bookLanguage = this.ttsEngine.opts.bookLanguage;
+        const bookLanguages = this.ttsEngine.getVoices().filter(v => v.lang.startsWith(bookLanguage)).sort(sortedOrder());
+        const otherLanguages = this.ttsEngine.getVoices().filter(v => !v.lang.startsWith(bookLanguage)).sort(sortedOrder());
+
         if (this.ttsEngine.getVoices().length > 1) {
-          voicesMenu.append(this.ttsEngine.getVoices().map(
-            voice =>
-              $(`<option value="${voice.voiceURI}">${voice.lang} - ${voice.name}</option>`)));
+          voicesMenu.append($(`<optgroup label="Book Language (en)"> ${renderVoiceOption(bookLanguages)} </optgroup>`));
```

Comment:
We will want the `if` to now be:

```js
if (bookLanguages.length > 1) { ... }
if (otherLanguages.length > 1) { ... }
```

to avoid rendering empty groups